quality-assurance/testing/simple_test_lung/README.md

```diff
@@ -14,33 +14,38 @@ BEFORE purchasing any parts.**
 
 ### Purchasing Source Key
 
-* **E** = ebay
 * **Z** = Amazon
-* **C** = McMaster-Carr
+* **R** = Made by RespiraWorks
 
 ### Parts
 
-| Item | Quantity | Manufacturer  | Part #         | Price (USD)     | Action | Sources         | Notes |
-| ---- |---------:| ------------- | ------------------- | ------------:|-----------|----------------| ----- |
-| 1    |        1 | Portex        | 370924              | 15.00        |*ask first*| search ebay     | test "lung" or breathing bag. |
-| 2    |        1 | South Pacific | SPB-6110-5PK        |   129.00 / 5 |*ask first*| [Z][2amzn]     | **ALT for item 1** |
-| 3    |        2 | Mars Wellness | 4332593260          | 16.99 / 2    | buy       | [Z][3amzn]     | CPAP tubing |
+| Item | Quantity | Manufacturer  | Part #                 | Price (USD)  | Sources        | Notes |
+| ---- |---------:| ------------- | ---------------------- | ------------:|----------------| ----- |
+| 1    |        1 | South Pacific | SPB-6110-5PK           | 129.00 / 5   | [Z][1amzn]     | Test lung |
+| 2    |        2 | Mars Wellness | 4332593260             | 16.99 / 2    | [Z][2amzn]     | CPAP tubing |
+| 3    |        1 | RespiraWorks  | patient_tubing_adapter | ~$1.0        | [R][3rw]       | Y-adapter for 22mm tubing |
 
-[2amzn]:  https://www.amazon.com/FlexLung-Biomedical-Ventilator-Testing-Demonstration/dp/B07B876P9C
-[3amzn]:  https://www.amazon.com/gp/product/B01N14F1MV
 
-**TODO:** pricing analysis
+[1amzn]:  https://www.amazon.com/FlexLung-Biomedical-Ventilator-Testing-Demonstration/dp/B07B876P9C
+[2amzn]:  https://www.amazon.com/gp/product/B01N14F1MV
+[3rw]: #tubing-adapter
 
-## Assembly
+#### Tubing adapter
+
+A "tee" or "why" adapter is needed to connect both the inhale and the exhale tubes to the single port on the test lung.
```

Comment:
this is usually called a "Y" adapter or a "wye" adapter but rarely a "why" adapter

Reply:
Changed